src/cpl/mct/rof_import_export.F90

```diff
@@ -21,6 +21,10 @@ module rof_import_export
   implicit none
   public
 
+  integer     ,parameter :: debug = 0  ! internal debug level
```

Comment:
Do we want to keep this extra debugging infrastructure? I'm not sure either way right now, but I want to make sure we think about the question.

Reply:
I think it's fine to leave this in. And it's in both drivers. So I'll just close this question.